tests/sentry/api/endpoints/test_organization_dashboard_widget_details.py

```diff
@@ -272,3 +272,57 @@ def test_invalid_issue_query_conditions_in_discover_widget(self):
         assert response.status_code == 400, response.data
         assert "queries" in response.data, response.data
         assert response.data["queries"][0]["conditions"], response.data
+
+    def test_raw_equation_in_orderby_is_valid(self):
```

Comment:
can we add a test case for the index format orderby `equation[0]` if there isn't one already? (there might already be, i haven't checked)

Reply:
https://github.com/getsentry/sentry/pull/33803/commits/a5aa3e60d5cb63f6fb8eae0692876d5965bfa3ce

I added more tests to cover the following cases:
* a valid equation and equation alias in the orderby
* an equation alias where the index is too high since there's one equation
* an equation alias where there are no equations

Hopefully these are what you had in mind. Let me know if otherwise!

Reply:
this is great, thanks!